extensions/two-factor-auth/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "two-factor-auth",
+  "title": "Two-Factor Auth",
+  "description": "Two-factor authentication for Raycast",
+  "icon": "command-icon.png",
+  "author": "cjdenio",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Two-Factor Auth",
```

Comment:
Since this name will be in search, do you think something like "Search 2FA Codes" or "2FA Codes" could be more searchable than typing "two-factor"? Don't have strong feelings about that one, up to you. You can also consider adding keywords to package.json.